pkg/platform/updog/updog.go

```diff
@@ -41,6 +41,8 @@ type executable struct {
 	log logging.SubLogger
 }
 
+// Boolean currently only used by CheckUpdate (and consequently ListUpdate). Returns true if the
```

Comment:
nit: conventionally, this is worded with `runOk` being the subject (at the start of the wording)

```go
// runOk executes the provided command, checking for printed output and error
// from the process. This command is used by CheckUpdate (and ListUpdate by
// extension) where printed output indicates that there is an update available.
func (e *executable) runOk(...) ... {
```